src/components/SubmitForm.vue

```diff
@@ -1,14 +1,17 @@
 <template>
-  <div id="data" class="center">
+  <div d="data" iclass="center">
     <form class="dataForm" @submit.prevent="addPerson">
-      <div class="q-pa-md">
-        <q-input name="firstName" v-model="firstName" color="primary" label="Introdu prenume" filled clearable />
```

Comment:
Strings such as labels shouldn't be directly in code, in general they should be in a string config file for translations
Try to add and configure i18n for your project
https://vue-i18n.intlify.dev/guide/installation.html#explanation-of-different-builds